views/doc/brakeman.rhtml

```diff
@@ -0,0 +1,9 @@
+<div class="">
+  <span class="sub-subtitle"><a href="#brakeman">Brakeman</a></span>
+  <ul class="mt--5">
+    <% group_brakeman_issues_by_warnings.each do |warning, issues| %>
```

Comment:
It would be good if we can do next:
```html
<% issues.tally.each do |type, count| %>
  <li>
    <%= "#{type}: #{count}" %>
  </li>
<% end %>

<span>
  <%= "There are #{issues.count} warnings and #{issues.high.count} of them are high. The full report can be found here." %>
</span>
```